src/dex-ui-components/base/Inputs/styles/RangeDatePickerStyles.ts

```diff
@@ -0,0 +1,31 @@
+import { Color } from "../../../themes";
+
+export const RangeDatePickerStyles = {
```

Comment:
The date picker styles are far off from the wireframes. I know we already agreed to leave some of the major parts of the date picker as is from the library but we should at least apply the appropriate colors and fonts.

Reply:
<img width="268" alt="Screenshot 2023-01-19 at 3 01 06 PM" src="https://user-images.githubusercontent.com/89677167/213406938-8d5b431b-151a-4262-b95f-370033986075.png">

Agree, In the documentation no where mentioned how to customise the picker fully though, I could find some `classes` of the picker in inspect element and applied some of the changes, the font sizes in Figma are too big in the UI I am decreased some of them let me know in case you want the exact sizes from the Figma.

Reply:
Sounds good. If necessary, we can make additional style changes in a subsequent task.